server/src/routes/trpc.ts

```diff
@@ -0,0 +1,57 @@
+import { initTRPC } from "@trpc/server";
+import { createExpressMiddleware } from "@trpc/server/adapters/express";
+import { z } from "zod";
+import { getDb } from "~/db/lib/get-db";
+
+const createContext = () => {
+  return {
+    ...getDb(process.env.DATABASE_URL!),
+  };
+};
+
+const t = initTRPC.context<typeof createContext>().create();
+
+const router = t.router;
+
+const trpcRouter = router({
+  getStudentById: t.procedure
+    .input(z.object({ student_id: z.string().uuid() }))
+    .query(async (req) => {
+      const { student_id } = req.input;
+
+      const result = await req.ctx.db
+        .selectFrom("student")
+        .where("student_id", "=", student_id)
+        .selectAll()
+        .executeTakeFirst();
+
+      return result;
+    }),
+
+  getAllStudents: t.procedure.query(async (req) => {
+    const result = await req.ctx.db.selectFrom("student").selectAll().execute();
+
+    return result;
+  }),
+
+  createStudent: t.procedure
```

Comment:
Please add a TODO to check for duplicates

Reply:
should we treat names as unique?

Reply:
Names aren't unique, but we can treat emails as unique.

Reply:
got it, wasn't sure if we were going to track student emails

oos for this pr so I'll add a todo for now